examples/README.md

```diff
@@ -0,0 +1,62 @@
+# Amplitude Python SDK
+
+There are some example apps of using Amplitude Python SDK.
+
+## Usage
+
+### Setup before execution
+
+1. Install Amplitude Python SDK
+   1. `pip install amplitude-analytics`
+2. Replace `YOUR API KEY` in example code with your amplitude project api key.
+
+### Run trackevent.py
+
+This example initialized a amplitude client and tracked an event, then triggered a call back function.
+
+`python trackevent.py`
+
+### Run flask example app
```

Comment:
As we add more examples a single README might get unwieldy. I think having a README per example like ampli-examples would make it easy to consume. The top level README can link to the individual examples.